packages/thirdweb/src/transaction/actions/sign-transaction.ts

```diff
@@ -1,11 +1,13 @@
-import type { TransactionSerializable } from "viem";
+import * as ox__Hash from "ox/Hash";
```

Comment:
curious why you dont just `import { Hash } from 'ox'` ?

Reply:
cjs won't treeshake properly if I do, have to import it this way

Reply:
really? ugh